src/main/java/uk/gov/hmcts/reform/sscs/service/PostponementRequestService.java

```diff
@@ -23,23 +23,24 @@
 @Service
 public class PostponementRequestService {
 
-    public void processPostponementRequest(SscsCaseData sscsCaseData, UploadParty uploadParty) {
+    public void processPostponementRequest(SscsCaseData sscsCaseData, UploadParty uploadParty, Optional<UploadParty> docUploadParty) {
```

Comment:
why not call the first uploadParty originalSender, like you've done in the other method to remove ambiguity